boards/google,hana

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+assert_driver_present armv8-pmu-driver-present armv8-pmu
+assert_device_present armv8-pmu-a53-probed armv8-pmu pmu_a53
+assert_device_present armv8-pmu-a72-probed armv8-pmu pmu_a72
+
+assert_driver_present cros-ec-debugfs-driver-present cros-ec-debugfs
+assert_device_present cros-ec-debugfs-probed cros-ec-debugfs cros-ec-debugfs.*
+assert_sysfs_attr_present cros-ec-debugfs-attr-console_log /sys/kernel/debug/cros_ec/console_log
+assert_sysfs_attr_present cros-ec-debugfs-attr-pdinfo /sys/kernel/debug/cros_ec/pdinfo
+
+assert_driver_present cros-ec-dev-driver-present cros-ec-dev
+assert_device_present cros-ec-dev-probed cros-ec-dev cros-ec-dev.*
+
+assert_driver_present cros-ec-lightbar-driver-present cros-ec-lightbar
+# assert_device_present cros-ec-lightbar-driver-present is expected to fail.
```

Comment:
Will this get fixed with a kernel patch?

Generally speaking, it's OK to have tests that fail initially and then start passing.  The tests that have always failed are not tracked as regressions and appear as "unknown".  So even if this gets fixed in next and mainline but fails on stable branches it's fine from a KernelCI perspective.

Reply:
This test will never pass on this device so I removed it.